app/scripts/modules/amazon/loadBalancer/configure/createLoadBalancer.html

```diff
@@ -1,4 +1,4 @@
-<v2-modal-wizard heading="Create New Load Balancer">
+<v2-modal-wizard heading="{{isAdd ? 'Add Load Balancer' : 'Create New Load Balancer'}}">
```

Comment:
In the deploy stage, we use 'Configure' instead of 'Add' - we should probably make this consistent.

If it's decided that this operation will only create a new load balancer (and fail if it already exists), we might want to use 'Add' to make it a little more clear that the operation is going to fail if there's already one with that name.